README.md

```diff
@@ -22,54 +22,48 @@ If the estimator does not support `partial_fit`, a warning will be shown saying
 
 ## Examples
 #### TuneGridSearchCV
-`TuneGridSearchCV` example. The dataset used in the example (MNIST) can be found [here](https://drive.google.com/file/d/1XUkN4a6NcvB9Naq9Gy8wVlqfTKHqAVd5/view?usp=sharing). We use this dataset to exemplify the speedup factor of `TuneGridSearchCV`.
+To start out, it’s as easy as changing our import statement to get Tune’s grid search cross validation interface, and the rest is almost identical!
 
 ```python
-from tune_sklearn import TuneGridSearchCV
-
-# Load in data
-from scipy import io
-data = io.loadmat("mnist_data.mat")
+# from sklearn.model_selection import GridSearchCV
+from tune_sklearn.tune_gridsearch import TuneGridSearchCV
```

Comment:
`from tune_sklearn import TuneGridSearchCV` should work now after PR #33

Reply:
Thanks, good catch